libs/template/config.go

```diff
@@ -125,12 +125,8 @@ func (c *config) skipPrompt(p jsonschema.Property, r *renderer) (bool, error) {
 		return false, nil
 	}
 
```

Comment:
We can add a function here to parse all properties in `p.Schema.SkipPromptIf.Properties` and `.Schema.SkipPromptIf.AnyOf...` and make those implicitly required by setting p.Schema.SkipPromptIf.Required = `[All keys accumulated`.

This adds an implicit requirement that any keys used in a `skip_prompt_if` condition are requried

Reply:
It's fine to directly mutate the schema by adding the `required` condition here since the schema in `SkipPromptIf` is not being used anywhere else.

Reply:
Kk added them to required!